lms/static/sass/discussion/views/_home.scss

```diff
@@ -0,0 +1,166 @@
+// discussion - home layout
+// ====================
+
+// home - layout
+.container .discussion-body {
+
+  .blank-slate {
+      padding: 40px 40px 10px;
```

Comment:
I know you're just moving, but can these values get the ``$baseline`` treatment?